apps/nuxt3-ssr/components/VariableCard.vue

```diff
@@ -11,7 +18,7 @@ defineProps<{
       <div class="grow flex items-center">
```

Comment:
Text that Overflows other text.
<img width="461" alt="Schermafbeelding 2023-10-20 om 10 59 54" src="https://github.com/molgenis/molgenis-emx2/assets/46395349/4a4ba682-7717-4617-be87-6eac51b2f948">

Reply:
issue in master , not part of this pr